recipes/javafx/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "javafx-sdk" %}
+{% set version = "11.0.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/javafxports/openjdk-jfx/archive/11.0.4+1.tar.gz
+  sha256: f03d16adae701406ff9e1fc05e461a327ca4f97b5bf18b0d727037bb425ec25a
+
+build:
+  number: 0
+  skip: true  # [win]
+  missing_dso_whitelist:
+    - '*'
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bison
+    - flex
+    - gperf
+    - pkgconfig
+    - gtk2
+    - pango
+    - freetype
+    - ant
+    - libx11-devel-cos6-x86_64
+    - libxxf86vm-devel-cos6-x86_64
+    - mesa-libgl-devel-cos6-x86_64
```

These look like linux only dependencies that should be used as CDTs?
```suggestion
    - {{ cdt('mesa-libgl-devel') }}  # [linux]
```

---

I'll try that.